src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/CodeActions/CodeActionEndpoint.cs

```diff
@@ -219,6 +220,16 @@ internal async Task<IEnumerable<RazorCodeAction>> GetCSharpCodeActionsFromLangua
 
             context.Request.Range = projectedRange;
 
+            // Ensures the end of the range > start of the range
+            // Should never occur, but was encountering this edge case.
+            if ((context.Request.Range.End.Line < context.Request.Range.Start.Line) ||
```

Comment:
@TanayParikh looks like this wasn't merged back into master? Is that expected or was this done differently in the master branch?

Reply:
This logic was moved to the `DefaultRazorDocumentMappingService` as part of https://github.com/dotnet/aspnetcore-tooling/pull/2576/files#diff-42d94fa8acbb00676d9fa61251875cd3R96.

Reply:
Alright cool, i'll exclude this